index.bs

```diff
@@ -560,76 +586,535 @@ message? -->
+APIs in [[REMOTE-PLAYBACK|Remote Playback API]] map to the protocol messages
+defined in this section.  
+ 
+For all messages defined in this section, see Appendix A for the full
+CDDL definitions.
+
+<!-- TODO: Add a capability that indicates support for the
+remote playback protocol.
+See https://github.com/webscreens/openscreenprotocol/issues/123 -->
+<!-- TODO: Frequency to update current-time (HtmlMediaElement does every 250ms?) -->
+<!-- TODO: extended mime types in source tag? -->
+<!-- TODO: media queries replacing URLs? -->
+
+To learn which receivers are [=compatible remote playback device=]s (also called
+available [=remote playback devices=]) for a particular URL or set of URLs, the
+controller may send a remote-playback-availability-request message with the
+following values:
+
+: urls
+:: A list of [=media resources=].  Must not be empty.
+
+: headers 
+:: headers that the receiver should use to fetch the
+    urls.  For example,
+    [[REMOTE-PLAYBACK#establishing-a-connection-with-a-remote-playback-device|section 6.2.4 of
+    the Remote Playback API]] says that the Accept-Language header should be
+    provided.
+
+: watch-duration
+:: The period of time that the controller is interested in receiving updates
+    about the URLs, should the availability change.
+
+: watch-id
+:: An identifier the receiver may use when sending updates about URL
+    availability so that the controller knows which URLs the receiver is referring
+    to.
+
+In response, the receiver should send a remote-playback-availability-response
+message with the following values:
+  
+: url-availabilities
+:: A list of URL availability states (available,
+    unavailable, or invalid).  Each state must correspond to the matching URL
+    from the request by list index.
+
+
+The receivers should later (up to the current time plus request
+watch-duration) send remote-playback-availability-event  messages if
+URL availabilities change.  Such events contain the following values:
+
+: watch-id
+:: The watch-id given in the presentation-url-availability-response,
+    used to refer to the presentation URLs whose availability has changed.
+
+: url-availabilities
+:: A list of URL availability states (available,
+    unavailable, or invalid).  Each state must correspond to the URLs from the
+    request referred to by the watch-id.
+
+Note that these messages are not broadcasted to all controllers. They are sent
+individually to controllers that have requested availability for the URLs that
+have changed in availability state within the watch duration of the original
+availability request.
+
+To save power, the controller may disconnect the QUIC connection and
+later reconnect to send availablity requests and receive availability
+responses and updates.
+
+
+To start remote playback, the controller may send a
+remote-playback-start-request message to the receiver with the following
+values:
+
+: remote-playback-id 
```

Comment:
It doesn't have any relationship to the presentation ID.  The presentation ID is more constrained because of the ability to join an existing presentation.